lua/cmake-tools/const.lua

```diff
@@ -114,5 +114,8 @@ local const = {
   },
   cmake_virtual_text_support = true, -- Show the target related to current file using virtual text (at right corner)
 }
-
+local osys = require("cmake-tools.osys")
+if osys.iswin32 then
+  const.cmake_build_directory = "out\\${variant:buildType}"
```

Comment:
If it is windows, and the path includes `/`, then replace `/` to `\\`.

Reply:
i think we should possibly add that replace to utils.transform_path later if there will be more errors like this